roles/common/tasks/netdata.yml

```diff
@@ -0,0 +1,65 @@
+---
+# install netdata on node
+- name: Download the installation script
+  get_url:
+    url: https://my-netdata.io/kickstart.sh
+    dest: ~/kickstart.sh
+    mode: +x
+
+- name: Install Netdata
+  command: ~/kickstart.sh --dont-wait
+
+- name: Cleanup installation script
+  file:
+    path: ~/kickstart.sh
+    state: absent
+
+- name: Set hostname variable
+  set_fact:
+    hostname: "{{ inventory_hostname | default('localhost') }}"
+
+# configure netdata
+- template:
+    src: ../templates/netdata.conf.j2
+    dest: /etc/netdata/netdata.conf
+    owner: root
+    group: root
+    mode: u=wrx,g=rx,o=r,+x
+  notify: Restart Netdata
+  become: true
+
+#claim node and add it to netdata cloud
+- name: Claim to Netdata Cloud
+  block:
+
+  - name: Claim to Netdata Cloud if not already
+    shell:
+      cmd: netdata-claim.sh -token={{ claim_token }} -rooms={{ claim_rooms }} -url={{ claim_url }}
+      creates: /var/lib/netdata/cloud.d/claimed_id
+    become: yes
+  when: reclaim == false
```

Comment:
Is there a way to check if its already claimed and skip this command altogether?